.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on: [push]
```

Comment:
You need to make this build on PRs as well like [here](https://github.com/profunktor/redis4cats/blob/master/.github/workflows/ci.yml#L7). Right now it's not building.